src/binary.ts

```diff
@@ -264,13 +269,20 @@ export class Binary extends BSONValue {
   }
 
   /** @internal */
-  [Symbol.for('nodejs.util.inspect.custom')](): string {
-    return this.inspect();
+  [Symbol.for('nodejs.util.inspect.custom')](
+    depth?: number,
+    options?: unknown,
+    inspect?: InspectParameterFn
+  ): string {
+    return this.inspect(depth, options, inspect);
   }
```

Comment:
Does it make sense to define this function (or switch them and then define `inspect()`) on `BSONValue`? `inspect()` is already declared as an abstract method there right now, and that would get rid of quite a bit of repetition.

Reply:
could you clarify what you mean by switching them?

Reply:
Mh, there are two methods here, `inspect()` and `[Symbol.for('nodejs.util.inspect.custom')]()`, where one just directly calls the other, and it doesn’t really matter which one calls which, so if one of them gets moved to the superclass, that could also be `[Symbol.for('nodejs.util.inspect.custom')]()` (i.e. keep things as is) or `inspect()` (i.e. switch around which method calls the other one).

Reply:
Sounds good, I removed  custom `[Symbol.for('nodejs.util.inspect.custom')]()` logic for all except regExp since it has a unique signature.

Reply:
It doesn’t have a unique signature, though, right? It doesn’t make use of the optional third argument but that’s still fine.